app/styles/layout/l-static-page.scss

```diff
@@ -2,4 +2,69 @@
 
 .l-static-page {
   padding: 85px 0 90px;
+
+  .section-page {
+
+    &::after {
```

Comment:
This is a duplication of https://github.com/Vizzuality/GlobalFishingWatch/blob/master/app/styles/layout/l-publications-articles.scss#L21
Please abstract into an util class.